src/main/java/com/salesforce/dev/framework/DataDrivenManager.java

```diff
@@ -56,7 +56,7 @@ public DataDrivenManager(){
         Collection<ViewSalesForce[]> viewSalesForcesArray =new ArrayList<ViewSalesForce[]>();
```

Comment:
I noticed that this class has similar methods
Could you refactor those methods?